apps/judicial-system/web/src/utils/stepHelper.ts

```diff
@@ -108,3 +111,22 @@ export const isTrafficViolationCase = (
       ),
   )
 }
+
+export const hasSentNotification = (
+  notificationType: NotificationType,
+  notifications?: Notification[],
+) => {
+  if (!notifications || notifications.length === 0) {
+    return false
+  }
+
+  const n = notifications.filter(
+    (notification) => notification.type === notificationType,
+  )
+
+  const latestN = n.sort((a, b) =>
```

Comment:
Notifications are returned in order, starting with the latest notification.

Consider removing the sort.